execution/aggregate/khashaggregate.go

```diff
@@ -100,7 +101,7 @@ func (a *kAggregate) Next(ctx context.Context) ([]model.StepVector, error) {
 	a.paramOp.GetPool().PutVectors(args)
 
 	if len(args) != len(in) {
-		return nil, fmt.Errorf("scalar argument not found")
+		return nil, errors.New("scalar argument not found")
```

Comment:
What is the advantage of using `errors.New` instead of `fmt.Errorf`?

Reply:
Also effort on Thanos to port entire codebase to it: https://github.com/thanos-io/thanos/pull/5382 (although now we might do it from efficientgo/core